WhatsApp/app/src/main/res/layout/item_call.xml

```diff
@@ -62,27 +62,16 @@
         android:orientation="vertical"
         app:layout_constraintGuide_percent="0.85" />
 
-    <com.google.android.material.imageview.ShapeableImageView
-        android:id="@+id/ivIcon"
-        android:layout_width="48dp"
-        android:layout_height="48dp"
-        android:layout_marginStart="12dp"
-        android:layout_marginTop="12dp"
-        android:layout_marginEnd="12dp"
-        android:layout_marginBottom="12dp"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toStartOf="@+id/guideline8"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:shapeAppearanceOverlay="@style/ShapeAppearanceOverlay.App.CornerSize50Percent"
-        app:srcCompat="@mipmap/ic_launcher" />
+    <include
+        android:id="@+id/ivProfileIcon"
```

Comment:
Need to refactor line 29 to reference new id

Reply:
thanks mate